elements/lisk-p2p/src/p2p.ts

```diff
@@ -727,6 +732,15 @@ export class P2P extends EventEmitter {
 				}
 
 				const existingPeer = this._peerPool.getPeer(peerId);
+				// Allow connections with lower version to have incoming connection even if it has outbound
+				if (
+					existingPeer &&
```

Comment:
can we also use `getUniqueOutboundConnections` and check it?

Reply:
we don't need that check because indirectly we are already checking it with `this._peerPool.getPeer(peerId)` if it exists or not